sqlx-core/src/error.rs

```diff
@@ -253,14 +253,6 @@ impl From<sqlx_rt::native_tls::Error> for Error {
     }
 }
 
-#[cfg(feature = "_tls-rustls")]
-impl From<webpki::InvalidDNSNameError> for Error {
-    #[inline]
-    fn from(error: webpki::InvalidDNSNameError) -> Self {
-        Error::Tls(Box::new(error))
-    }
-}
-
```

Comment:
@paolobarbolini isn't this technically a breaking change?

Reply:
It is :disappointed:

Reply:
I've got this slated for 0.6.0 then. Currently trying to get a 0.5.10 out though.